README.md

```diff
@@ -1,2 +1,15 @@
 # Seine-HelloWorld
 The testing repo for Seine Bot
+
+
+# Assignment 1 (5/19/24) Test Event Types for Seine AI
+
+1. Fork this repository. Create/Edit a file of your choice.
+2. Create a PR with your commmited changes
```

Comment:
@SeineSailor test redirect event

Reply:
@SeineSailor test redirect event

Reply:
@SeineSailor I fixed workflow_filename parsing error, let's try again

Reply:
@SeineSailor fixed bash format error. try again

Reply:
@SeineSailor fixed header. try again

Reply:
@SeineSailor

debug why github.event.id is empty

change ref branch to main

Reply:
@SeineSailor

dump whole github payload to debug

Reply:
@SeineSailor github event id cannot be accessed during a run. pass event name and payload instead.

Reply:
test whether hardcoding the url works @SeineSailor

Reply:
set actions:write @SeineSailor

Reply:
try rabbit